client/src/components/PhotoModalCarousel.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Photo from './Photo.jsx';
+
+const PhotoModalCarousel = (props) => (
+    <div className="photoModalCarousel">
+      {props.photos.map((photo, i) => {
+        if (i === props.currentPhoto) {
+          return <Photo photoClass={'carouselPhoto'} photoUrl={photo} key={i}/>
+        } else {
+          return <Photo photoClass={'carouselPhotoActive'} photoUrl={photo} key={i}/>
+        }
```

Comment:
Wouldn't it be nice to handle the if/else business first, then instantiate a `<Photo />` component as the next step, setting the `photoClass` prop to a variable instead of a hardcoded string?

```
{props.photos.map((photo, i) => {
  const photoClass = (i === props.currentPhoto) ? 'carouselPhoto' : 'carouselPhotoActive';
  return (
    <Photo 
      key={i}
      photoClass={photoClass}
      photoUrl={photo} 
    />
  );
})}
```

You determine if you wanna break up the instantiation into several lines or not, up to you, I can't see the character count from this view. /shrug